ament_cmake_clang_format/cmake/ament_clang_format.cmake

```diff
@@ -15,15 +15,21 @@
 #
 # Add a test to check the code for compliance with clang_format.
 #
+# The configuration file for clang-format to use can be either set by
```

Comment:
> ... can be either set by ...

Proposing: ... can be either overridden by ...

Maybe add a sentence before that mentions the configuration file used by default?

Reply:
Just added the suggested change along with a sentence specifying the default configuration file